frontend/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:16
+WORKDIR /app
```

Comment:
Any reason of not putting it under /usr/src/app like other Dockerfile?

Reply:
After reading several articles on dockerizing React apps, I believe `/app` is the convention.